README.md

```diff
@@ -1,11 +1,31 @@
-# eSchool
-Microservice based eSchool Solution
+# eSchool - Microservice and Containers based open source School Administration Software
 
 ## Linux Build Status for 'master' branch
 
 |                                                                                                    Identity API                                                                                                    |
 | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | [![Build Status](https://dev.azure.com/OpenCodeFoundation/eSchool/_apis/build/status/identity?branchName=master)](https://dev.azure.com/OpenCodeFoundation/eSchool/_build/latest?definitionId=1&branchName=master) |
 
+## Tech & Tools
+- .NET Core
+- Docker
+- Orchestrators: Kubernetes
+- Visual Studio
+- MongoDB
+- SQL Server
+- Azure DevOps
+- Redis
+- RabbitMQ
```

Comment:
I think we should add tools and technology as we work on it. Not before hand. But on the other hand, list of technology we are going to use is also good to let everyone know about our plan